workloads/scale-perf/run_scale_fromgit.sh

```diff
@@ -54,6 +69,7 @@ spec:
       serviceaccount: scaler
       poll_interval: $_poll_interval
       post_sleep: $_post_sleep
+$(_add_rosa_params)
```

Comment:
This is a bit tricky, I think you should consider using an if else logic to create the Benchmark CRDs

Reply:
It is the only way to add conditionals to a cat EOF structure without replicating all the content

Reply:
what do you think now?